server/conf/messages

```diff
@@ -318,8 +322,8 @@ content.confirmed=Thank you for applying to {0}.  Your application has been rece
 # Title (not a main page title) on section prompting an applicant to create an account or sign in to save their data.
 title.createAnAccount=Create an account or sign in
 
-# A message urging users to create an account to save their data.
-content.pleaseCreateAccount=If you sign into a city account now, your information will be saved and you''ll be able to return at any time to complete future applications.  If you do not already have an account, the login page will allow you to register.  You won''t lose any data you''ve already entered.
+# Shown to applicants to encourage login; {0} represents the civic entity's name, in reference to their login portal.
+content.pleaseCreateAccount=Log in to your {0} account or create an account to have your information saved. You''ll be able to return at any time to complete future applications. If you do not already have an account, the login page will allow you to sign up and you won''t lose any data you''ve already entered.
```

Comment:
```suggestion
content.pleaseCreateAccount=Log in to your {0} account or create an account to have your information saved. You'll be able to return at any time to complete future applications. If you do not already have an account, the login page will allow you to sign up and you won't lose any data you've already entered.
```

Reply:
We need the double apostrophes:
https://github.com/civiform/civiform/blob/8b443012c970ab8c46816548906f81b39869155b/server/conf/messages#L5